contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -248,6 +248,11 @@ Start a terminal, go to the root dir of the engine source code and type:
     ``linuxbsd``. If you are looking to compile Godot 3.x, make sure to use the
     `3.x branch of this documentation <https://docs.godotengine.org/en/3.6/development/compiling/compiling_for_x11.html>`__.
 
+If you wish to add and use breakpoints while developing, add dev_build=yes
```

Comment:
```suggestion
If you wish to add and use breakpoints while developing, add ``dev_build=yes``
```

Reply:
`dev_build` enable additional runtime checks and disables optimization, it will make debugging more precise, but is not strictly required to debug/set brakepoints (with `debug_symbols=yes` it can be done in optimized build as well).

`dev_mode` is not related to debugging at all, it's enabling extra check during engine build.